daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/TypeCalculator.scala

```diff
@@ -153,7 +153,7 @@ class KeysetValueTypeCalculatorOrdered(valueMap: HashMap[DataValuePrimitive, Dat
       })
       ans1 match {
         case None => {
-          (DataValue.NoValue, One(s"Key ${x} not found in enumeration key-value mapping"))
+          (DataValue.NoValue, One(s"Key ${x} not found in enumeration dfdlx:repValues keyset-value mapping"))
```

Comment:
I suggest drop the words Key and Keyset entirely, if you want them in the message for compatiblity reasons add literally "\n(formerly keyset-value mapping)" at the end of the message, but don't otherwise use this terminology.

Reply:
I suggest we just change tests to remove `keyset` or `mapping` if any look for those words instead of including the "formerly..." wording in the error message. We don't guarantee compatibility with error messages for users, and that part of the error message is confusing and probably isn't worth keeping for compatibility.